api/app/src/models/db.ts

```diff
@@ -24,7 +23,6 @@ const models: ModelSetup[] = [
   PatientModel.setup,
   MAPIAccess.setup,
   CustomerSequenceModel.setup,
-  DocumentReferenceModel.setup,
```

Comment:
I see we're not deleting the table now, which I think makes sense. Maybe create a ticket to remove it later?

Reply:
https://github.com/metriport/metriport-internal/issues/605